bridge-fungible-token/src/utils.sw

```diff
@@ -37,14 +36,15 @@ fn shift_decimals_left(bn: U256, d: u8) -> Result<U256, BridgeFungibleTokenError
     // shift decimals in increments of the max power of 10 that bn_mult will allow (10^19)
     while (decimals_to_shift > 19) {
         // note: 10_000_000_000_000_000_000 = 10.pow(19)
-        let (adjusted, overflow) = bn_mult(bn_clone, 10_000_000_000_000_000_000);
+        let (adjusted, overflow) = bn_mult(bn, 10_000_000_000_000_000_000);
         if (overflow != 0) {
             return Result::Err(BridgeFungibleTokenError::OverflowError);
         };
         decimals_to_shift = decimals_to_shift - 19;
         bn_clone = adjusted;
```

Comment:
Should this be just `bn` and we get rid of `bn_clone`? Whatever is being used needs to be what gets multiplied on line 47

Reply:
I'll revisit this. Some of these changes involving `bn_clone` were made to try to fix a transient bug, but I may have not switched them all back to the way they were originally.
That said, all the tests are passing, so it's either working fine or needs more test coverage. My hunch is the later.

Reply:
I've opted to keep `mut bn_clone` as we need to  mutate it in the function body. If I try to use `bn` directly, I need to make the param `ref mut`, and then make any values passed to the function as this param mutable as well, which I don't love.
I've cleaned this up though so the usage is consistent in both `shift_decimals...` functions.